ush/gfs_bufr.sh

```diff
@@ -19,30 +19,18 @@
 # 2019-10-10  Guang Ping Lou: Read in NetCDF files
 # echo "History: February 2003 - First implementation of this utility script"
 #
-source "$HOMEgfs/ush/preamble.sh"
+source "${HOMEgfs:-?}/ush/preamble.sh"
```

Comment:
Is there a difference between `:-?` and `:?`?

Reply:
Actually, `:?` is correct. I need to change these.

Reply:
Fixed